src/SettingsSectionAccount/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import classNames from 'classnames';
+import SectionHeaderGeneral from '../SectionHeaderGeneral';
+import SettingsItemEmail from '../SettingsItemEmail';
+import SettingsItemPassword from '../SettingsItemPassword';
+import moment from 'moment';
+import Wrapper from '../Wrapper';
+import Typography from '../Typography';
+import Spacer from '../Spacer';
+import styles from './styles.css';
+const PropTypes = React.PropTypes;
+
+function SettingsSectionAccount(props) {
+  return (
+    <div className={styles.outerWrapper}>
+      <SectionHeaderGeneral
+        title={props.title}
+        />
+      <SettingsItemEmail
+        email={props.email}
+        emailOnClick={props.emailOnClick}
```

Comment:
`emailOnClick={props.emailOnClick}` wont work. On the left of the equals we need the name of the prop in the parent component and on the right we can give the prop a new name for this component. 
So in this case we say `onClick={props.emailOnClick}` since the parent component is expecting to receive something on `onClick`